src/test_l1_tvSettings.c

```diff
@@ -12716,9 +12684,10 @@ void test_l1_tvSettings_positive_GetSupportedComponentColor (void)
 
 	/* Step 01: Calling tvsettings initialization and expecting the API to return success */
 	result = TvInit();
-	UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
+	UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);	
+
 
-	/* Step 02: Calling tvsettings GetSupportedComponentColor and expectinging the API to return success */
+	/* Step 03: Calling tvsettings GetSupportedComponentColor and expectinging the API to return success */
```

Comment:
updating the steps number can be reverted

Reply:
reverted